test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_DeleteConsumerGroupOffsets.cs

```diff
@@ -0,0 +1,128 @@
+
+                committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(Offset.Unset, committedOffsets[0].Offset);
+
+                // Consumer is actively subscribed to the topic
+                consumer1.Subscribe(new List<String>(){topic1.Name});
+                consumer1.Commit(new List<TopicPartitionOffset>() {new TopicPartitionOffset(topic1.Name, 0, offsetToCommit)}); //commit some offset for consumer
+
+                committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                topicPartitionToReset = new List<TopicPartition>() {new TopicPartition(topic1.Name, 0)};
+                consumerGroupOffsetsToDelete = new List<ConsumerGroupOffsets>() {new ConsumerGroupOffsets(groupId1, topicPartitionToReset)};
+                res = adminClient.DeleteConsumerGroupOffsetsAsync(consumerGroupOffsetsToDelete).Result;
+                Assert.Equal(1, res.Count);
+                Assert.Equal(groupId1, res[0].Group);
+                Assert.Equal(1, res[0].Partitions.Count);
+                Assert.Equal(0, res[0].Partitions[0].Partition.Value);
+
+                committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(10, committedOffsets[0].Offset); //offset is unchanged as the consumer is actively subscribed to the topic
+
+                consumer1.Unsubscribe();
+
+                // Deleting offset from the topic which has no set offset
+                consumer2.Assign(new List<TopicPartition>(){new TopicPartition(topic2.Name, 0)});
+
+                committedOffsets = consumer2.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(Offset.Unset, committedOffsets[0].Offset);
+
+                topicPartitionToReset = new List<TopicPartition>() {new TopicPartition(topic2.Name, 0)};
+                consumerGroupOffsetsToDelete = new List<ConsumerGroupOffsets>() {new ConsumerGroupOffsets(groupId1, topicPartitionToReset)};
+                res = adminClient.DeleteConsumerGroupOffsetsAsync(consumerGroupOffsetsToDelete).Result;
+                Assert.Equal(1, res.Count);
+                Assert.Equal(groupId1, res[0].Group);
+                Assert.Equal(1, res[0].Partitions.Count);
+                Assert.Equal(0, res[0].Partitions[0].Partition.Value);
+
+                committedOffsets = consumer2.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(Offset.Unset, committedOffsets[0].Offset); //offsets are unchaged after the reset
+                
+                // Deleting offsets for more than one group at a time results in an exception
```

Comment:
this shouldn't fail - you should be able to delete from more than one group at a time.

Reply:
suggest setting `Debug` to `all` and looking at the librdkafka output to try and figure out why this is failing. 

the problem is i can't remember how to capture stderr logging from XUnit tests, or even whether i managed to do it. try and research that, if you fail, you can make a console application with this logic and test with that.

i've considered not using Xunit and just doing tests as a console app. the XUnit test framework doesn't add much, except headaches in trying to work out how to do things like the above in my experience.

Reply:
I searched the error in librdkafka and looks like librdkafka is explicitly allowing only a single group. https://github.com/edenhill/librdkafka/blob/75a243f0e41e3566d9398a29e4abfb8139c7e84f/src/rdkafka_admin.c#L3932-L3941

Reply:
Removed this test as it wasn't needed with the current API which only allows a single group at a time.